crates/epaint/src/text/text_layout.rs

```diff
@@ -400,6 +414,7 @@ fn replace_last_glyph_with_overflow_character(
     fonts: &mut FontsImpl,
     job: &LayoutJob,
     row: &mut Row,
+    pixels_per_point: f32,
```

Comment:
nit: egui tries to order its arguments in a logical way, with the first argument being "context-like", e.g. things that are passed down through many hierarchies unchanged (in this case: `fonts, pixels_per_point`), while the latter arguments are the the things that are more likely to be specific to the function (in tis case: `row`)